logos/logos-ui/src/app/features/agents/agents.scss

```
@@ -0,0 +1,442 @@
+  dt {
+    font-size: var(--font-size-xs);
+    text-transform: uppercase;
+    letter-spacing: var(--letter-spacing-widest);
+    color: var(--color-typography-300);
+  }
+
+  dd {
+    margin: 0.2rem 0 0;
+    font-size: var(--font-size-sm);
+    color: var(--color-typography-700);
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)